### ISSUE TITLE
update to re-align with core DID spec

### ISSUE THREAD

**Issue by dhh1128** [decentralized-identity/peer-did-method-spec#5]
The peer DID method was written before many recent evolutions in the DID spec. The `authorization` section of a peer DID doc used to be aligned with the DID spec, but it no longer is. We need to recast this as an extension, and/or decide to eliminate it in some way.

There may be other details to re-align as well.

**Comment by jmandel** [comment]
Agreed this would be helpful! I'm trying to cross-walk and find myself needing to imagine/invent details to make the peer DID examples valid.
